### PR TITLE
Added information about community edition

### PR DESCRIPTION
**Pull request by DJ1TJOO**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ We are currently working on a successor to shapez, with 3D Graphics, Exploration
 
 <br>
 
+# Shapez Community Edition
+Shapez Community Edition is a community maintained version of shapez.
```
Comment (EmeraldBlock) [inline README.md:12]: hyphenate "community-maintained"

File: README.md [hunk 2]
```diff
@@ -8,6 +8,9 @@ We are currently working on a successor to shapez, with 3D Graphics, Exploration
 
 <br>
 
+# Shapez Community Edition
```
Comment (EmeraldBlock) [inline README.md:11]: "shapez" is lowercase

File: README.md (hunk 1 above)
Comment (EmeraldBlock) [inline README.md:12]: perhaps mention the abbreviation "shapez CE"?

File: README.md [hunk 3]
```diff
@@ -8,6 +8,9 @@ We are currently working on a successor to shapez, with 3D Graphics, Exploration
...
+Shapez Community Edition is a community maintained version of shapez.
+
 # shapez
```
Comment (EmeraldBlock) [inline README.md:14]: just inserting a new section makes for a rough transition

File: README.md [hunk 4]
```diff
@@ -8,6 +8,9 @@ We are currently working on a successor to shapez, with 3D Graphics, Exploration
 
 <br>
 
+# Shapez Community Edition
+Shapez Community Edition is a community maintained version of shapez.
+
```
Comment (EmeraldBlock) [inline README.md:13]: need more details!!!

**Review by EmeraldBlock — CHANGES_REQUESTED** [review]
me picking apart the 2-line change

**Comment by DJ1TJOO** [comment]
> me picking apart the 2-line change

This was just more so there is something about it on the page. Else people get to the repo and don't know what's going on

**Review by EmeraldBlock — APPROVED** [review]
rewrote README; bagel and dengr approved

**Pull request merged**